hypothesis-python/src/hypothesis/strategies/_internal/numbers.py

```diff
@@ -290,9 +337,11 @@ def floats(
       allow_nan.
     - If both min_value and max_value are not None, it is an error to enable
       allow_infinity.
+    - If inferred values range does not include subnormal values, it is an error
+      to enable allow_subnormal.
 
-    Where not explicitly ruled out by the bounds, all of infinity, -infinity
-    and NaN are possible values generated by this strategy.
+    Where not explicitly ruled out by the bounds, all of the subnormals,
+    infinity, -infinity and NaN are possible values generated by this strategy.
```

Comment:
I think subnormal values are rare enough that we should probably include e.g. a `:wikipedia:` link to explain them.